packages/frontend-common/formats/chart/network/NetworkBase.tsx

```diff
@@ -289,13 +292,46 @@ export const NetworkBase = ({
         );
     }, [nodes, highlightedNodeIds, selectedNode, hoveredNode]);
 
+    const [minLinkSize, maxLinkSize] = useMemo(() => {
+        const sizes = links
+            .map((l) => l.value)
+            .filter((v): v is number => typeof v === 'number');
+        return [Math.min(...sizes), Math.max(...sizes)];
+    }, [links]);
+
+    const linkColor = useLinkColor({
```

Comment:
NitPick: A Hook is overkill IMO, you could just have a function declared outside of the component.